velox/docs/functions/presto/math.rst

```diff
@@ -55,6 +55,13 @@ Mathematical Functions
 
     Returns the value of ``string`` interpreted as a base-``radix`` number. ``radix`` must be between 2 and 36.
 
+.. function:: inverse_laplace_cdf(mean, scale, p) → double
```

Comment:
This function has to be documented with the rest of the inverse probability functions in Probability Functions : inverse_cdf portion of this file.